packages/ui-markdown-editor/src/plugins/withText.js

```diff
@@ -10,8 +11,16 @@ export const withText = (editor) => {
 		if(isBlockHeading(editor)){
 			return;
 		}
-		const currentLine = currentNode.text;
-		matchCases(editor, currentLine);
+
+		const currentLine = currentNode.text
+
+		onkeyup = (ev) => {
+			switch (ev.key) {
+  	    case SPACE_BAR:
+					matchCases(editor, currentLine);
+				break;
+			}
+		}	
```

Comment:
Do you expect there to be other cases? If not, I think we could do this without a switch case to be more simple.

Reply:
I changed it to a simple if-else statement as I don't think there will be more cases for it right now.